src/ansys/dpf/core/operators/math/elemental_density.py

```diff
@@ -0,0 +1,343 @@
+"""
+elemental_density
+=================
+Autogenerated DPF operator classes.
+"""
+
+from warnings import warn
+from ansys.dpf.core.dpf_operator import Operator
+from ansys.dpf.core.inputs import Input, _Inputs
+from ansys.dpf.core.outputs import Output, _Outputs
+from ansys.dpf.core.operators.specification import PinSpecification, Specification
+
+
+class elemental_density(Operator):
+    """Extract Elemental Topology Density result from topo solver output.
+    Default behavior is to use graphical density.
+
+    Parameters
+    ----------
+    time_scoping : Scoping, optional
+    mesh_scoping : Scoping, optional
+    streams : StreamsContainer, optional
+        Topo file stream.
+    data_sources : DataSources, optional
+        Topo file data source.
+    custom_ponderation_name : str
+        Take custom ponderation_field from the topo
+        file by name
```

Comment:
Brief description should not begin with a verb (Take) and should use plain English. This descripton doesn't seem adequate.